modules/i18n/src/translations/en-US/portals/custom-auth-connection.ts

```diff
@@ -0,0 +1,144 @@
+                                required: "Header is a required field."
+                            }
+                        },
+                        password: {
+                            label: "Password",
+                            placeholder: "Password",
+                            validations: {
+                                required: "Password is a required field."
+                            }
+                        },
+                        username: {
+                            label: "Username",
+                            placeholder: "Username",
+                            validations: {
+                                required: "Username is a required field."
+                            }
+                        },
+                        value: {
+                            label: "Value",
+                            placeholder: "Value",
+                            validations: {
+                                required: "Value is a required field."
+                            }
+                        }
+                    },
+                    hint: "Once added, these secrets will not be displayed. You will only be able to reset them.",
+                    label: "Authentication Scheme",
+                    placeholder: "Select Authentication Type",
+                    title: "Endpoint Authentication",
+                    validations: {
+                        required: "Authentication Type is a required field."
+                    }
+                },
+                endpoint: {
+                    hint: "The URL of the configured external endpoint to integrate with the authenticator",
+                    label: "Endpoint",
+                    placeholder: "https://abc.external.authenticator/authenticate",
+                    validations: {
+                        empty: "Empty endpoint URI",
+                        general: "Please enter a valid URL.",
+                        invalid: "The entered URL is not HTTPS. Please add a valid URL."
+                    }
+                },
+                title: "Configuration"
+            },
+            generalSettingsStep: {
+                displayName: {
+                    hint: "",
+                    label: "Display Name",
+                    placeholder: "ABC Authenticator",
+                    validations: {
+                        empty: "",
+                        invalid: "Invalid Display Name"
+                    }
+                },
+                helpPanel: {
+                    displayName: {
+                        description: "Provide a meaningful name to identify the connection.",
+                        header: "Display Name",
+                        hint: "Must be a string containing only letters (a-z, A-Z) numbers (0-9), spaces, " +
+                        "underscore (_) and hyphen (-). "
```

Comment:
`string` is a technical word for a word (or a set of characters). Let's not use technical terms in the UI text unless absolutely necessary.

Please check and update the other places as well

```suggestion
                        hint: "The display name can contain only letters (a-z, A-Z), numbers (0-9), spaces, " +
                        "underscore (_) and hyphen (-). "
```

Reply:
These has been agreed upon prior to starting the development.

**Email Subject**: 
- [Architecture] Facilitate user authentication via a external service (custom authentication extension)

Reply:
> `string` is a technical word for a word (or a set of characters). Let's not use technical terms in the UI text unless absolutely necessary.
> 
> Please check and update the other places as well

would like to know your opinion on this as well. @brionmario @NipuniBhagya